### ISSUE TITLE
Cannot tab/give focus on adaptive card buttons using keyboard navigation

### ISSUE THREAD

**Issue by owenpww** [microsoft/BotFramework-WebChat#4112]
## Screenshots
![image](https://user-images.githubusercontent.com/93670080/150269883-f413d34e-002b-4d0c-9322-3ffa0ffd9d5c.png)

## Version
Using NPM package (React).
`'<meta name="botframework-directlinespeech:version" content="4.14.1">\n<meta name="botframework-webchat:bundle:variant" content="full">\n<meta name="botframework-webchat:bundle:version" content="4.14.1">\n<meta name="botframework-webchat:core:version" content="4.14.1">\n<meta name="botframework-webchat:ui:version" content="4.14.1">'`

## Describe the bug
I am trying to navigate through my entire page and therefore also the bot on the page using only the keyboard. However, I can't focus on the buttons inside the chat itself and I suspect any other elements without having to use the mouse. If I use the keyboard only then it focuses on the body of the chat then moves on to the input field, send button then outside of the React WebChat component to other elements on our page without focusing any of the elements inside of the chat.

## Steps to reproduce
1. Create react app to embed bot in.
2. Tab through entire page using keyboard only.
3. Confirm focus is never set on elements inside of chat even if they can receive focus when using the mouse.

## Expected behavior
For focus to be received on the accessible/focusable elements inside of the chat before focus moves out of the React WebChat component.

## Additional context

[Bug]

**Comment by owenpww** [comment]
My mistake. Pressing `enter` while the parent `div` is in focus allows focus to be received by the child elements. I assumed it would have given focus to the children without having to press `enter`.
